docs/guides/migrations.md

```diff
@@ -7,6 +7,15 @@ kb_sync_latest_only
 
 # Migrations
 
+## 4.1 to 4.2
+
+A better handling for cookie `SameSite` and `secure` settings was implemented with new defaults to `SameSite=Strict` and `secure`.
+This can still be overridden when calling `cookies.services` `put` method with explicitly set values.
+Additionally if the PWA is run in an iframe all cookies are set with `SameSite=None` and if run with `http` (should only be in development environments) `secure` is not set.
+For migrating the occurrence of `cookies.services` `put` can be check whether they need to be adapted.
+Before by default no `SameSite` was set so browsers treated it as `SameSite=Lax`, this needs to be set explicitly now if it is really intended.
+The `secure` setting was only set if explicitly set to `secure: true` and if in `https` mode otherwise it was set to `false`.
```

Comment:
Now the `secure` setting is always set to true if in `https` mode, you can prevent this by explicitly set it to 'false'.